jcache/commons/pom.xml

```diff
@@ -0,0 +1,42 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-jcache-parent</artifactId>
+      <version>7.2.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>infinispan-jcache-commons</artifactId>
+   <name>Infinispan JCACHE (JSR-107) Common Components</name>
+   <description>Common components for the JCACHE (JSR-107) implementations using Infinispan</description>
+
+   <dependencies>
+      <dependency>
+         <groupId>${project.groupId}</groupId>
+         <artifactId>infinispan-core</artifactId>
```

Comment:
Replaced this with a dep on commons.

Split logging.Log to share to avoid duplication and at the same time don't depend on core. One ugly side-effect though is that the _log ids_ are now split between two interfaces (one in commons, one in embedded). Do you have any suggestion on how to make this look less ugly?